### PR TITLE
update political district pages (removing outdated pages)

### PR DESCRIPTION
**Pull request by gregbunce**
WIP: making sure I'm on the right path.

### CODE REVIEW

File: _posts/2012-05-02-new-utah-political-districts-and-voting-precincts-gis-datasets.md [hunk 1]
```diff
@@ -15,7 +15,7 @@ categories:
 ### New Utah Political Districts
 {: .text-left}
 
-[Political districts for state-level offices]({% link data/political/2012-2022-political-districts/index.html %}) (Congressional, State House, State Senate, and State Board of Education) are available as a GIS data layers. These districts are for elections in 2012 and ongoing representation after the new representatives are installed on Jan 1, 2013.
+[Political districts for state-level offices]({% link data/political/index.html %}) (Congressional, State House, State Senate, and State Board of Education) are available as a GIS data layers. These districts are for elections in 2012 and ongoing representation after the new representatives are installed on Jan 1, 2013.
```
Comment (steveoh) [inline _posts/2012-05-02-new-utah-political-districts-and-voting-precincts-gis-datasets.md:18]: **question**: is the last sentence of this still accurate? Oh maybe it was in 2012...

**chore**: link to the 2012 shelved data

File: _posts/2014-08-26-large-format-utah-political-district-maps-now-available.md [hunk 2]
```diff
@@ -17,7 +17,7 @@ categories:
...
 <p>The maps are accessible from the <a href="https://elections.utah.gov/map/district-maps">Lt. Governor's Election Districts interactive web map</a>, which also allows custom map views and access to 8.5 x 11 inch district maps. Alternatively, the files, can be accessed from the list below. </p>
-<p>Lastly, a reminder that the GIS-formatted data for all statewide political districts is available for download from the <a href="{% link data/political/2012-2022-political-districts/index.html %}">SGID data pages</a>.</p>
+<p>Lastly, a reminder that the GIS-formatted data for all statewide political districts is available for download from the <a href="{% link data/political/index.html %}">SGID data pages</a>.</p>
```
Comment (steveoh) [inline _posts/2014-08-26-large-format-utah-political-district-maps-now-available.md:20]: **chore**: link to the 2012 data

File: _posts/2015-04-29-updates-to-political-districts-voting-precincts-datasets-april-2015.md [hunk 3]
```diff
@@ -9,7 +9,7 @@ categories:
   - SGID Blog
 ---
 <p><strong>Utah Political Districts:</strong></p>
-<p><a href="{% link data/political/2012-2022-political-districts/index.html %}">Political districts for state-level offices</a> (Congressional, State House, State Senate, and State Board of Education) are available as a GIS data layers. These districts are for elections and ongoing representation.</p>
+<p><a href="{% link data/political/index.html %}">Political districts for state-level offices</a> (Congressional, State House, State Senate, and State Board of Education) are available as a GIS data layers. These districts are for elections and ongoing representation.</p>
```
Comment (steveoh) [inline _posts/2015-04-29-updates-to-political-districts-voting-precincts-datasets-april-2015.md:12]: **chore**: link to the 2012 data

File: data/political/2012-2022-political-districts/index.html [hunk 4]
```diff
@@ -1,146 +0,0 @@
----
```
Comment (steveoh) [inline data/political/2012-2022-political-districts/index.html:1]: **chore**: undelete and mark as publish: false

File: data/political/2002-2011-house-senate-congressional-districts/index.html [hunk 5]
```diff
@@ -1,116 +0,0 @@
----
```
Comment (steveoh) [inline data/political/2002-2011-house-senate-congressional-districts/index.html:1]: **chore**: undelete and mark as publish: false

File: data/political/2022-2032-political-districts/index.html [hunk 6]
```diff
@@ -76,6 +76,8 @@ <h5>Related Resources</h5>
       <ul class="dotless">
```
Comment (steveoh) [inline data/political/2022-2032-political-districts/index.html:76]: **thought**: should we rename this to political districts so we don't have to modify it every time?

File: data/political/2022-2032-political-districts/index.html [hunk 7]
```diff
@@ -144,6 +149,11 @@ <h3 id="UnitedStatesCongressionalDistricts">United States Congressional District
       <p>U.S. Congress Districts 2022 to 2032 shows the U.S. Congressional Districts that elected officials represent.</p>
       <p>The <code>DISTRICT</code> field contains the U.S. Congress district number.</p>
       <p>Statewide Political District Boundaries are drawn by the Utah Legislature and adopted into state law.</p>
+	<h5>Related Resources</h5>
+      	<ul class="dotless">
+		<li><a href="https://utah.hub.arcgis.com/datasets/f86be475250b474d965e42971f50c0d3/explore">Previous Data: Utah US Congressional Districts 2012 to 2022</a></li>
```
Comment (steveoh) [inline data/political/2022-2032-political-districts/index.html:154]: **quibble**: there's a mixture of tabs and spaces. the tabs should be removed

**Review by steveoh — COMMENTED** [review]
this looks good. I have a few suggestions and a request. 

For the pages that we are unpublishing, we should create redirects to the current page.

File: data/political/2022-2032-political-districts/index.html (hunk 6 above)
Comment (gregbunce) [reply data/political/2022-2032-political-districts/index.html:76]: I like what you're saying.  it's gonna blow up some stuff with existing links on this site and also the sgid index.  Are you up for a little adventure (with the sgid index spreadsheet again and a fetch for my PR)?

File: data/political/2012-2022-political-districts/index.html (hunk 4 above)
Comment (gregbunce) [reply data/political/2012-2022-political-districts/index.html:1]: is there an easy way to do this, aka: remove the associated commit or should i just add the page back into this branch?

Comment (steveoh) [reply data/political/2012-2022-political-districts/index.html:1]: removing the commit would be the easiest way

File: data/political/2022-2032-political-districts/index.html (hunk 6 above)
Comment (steveoh) [reply data/political/2022-2032-political-districts/index.html:76]: Yah, certainly

File: data/political/2012-2022-political-districts/index.html (hunk 4 above)
Comment (gregbunce) [reply data/political/2012-2022-political-districts/index.html:1]: Is there an easy way to remove a commit?  I'm seeing that option in the browser.

**Comment by gregbunce** [comment]
I'll pick this back up on Friday.  I'm out of the office tomorrow.  

we'll have to coordinate the sgid index stuff, though, so I'll hold off on that piece until you are back.

also, the main blocker for me at the moment is how to revert those two commits (that removed the two older pages).  I can't seem to find how to do it in the browser. I could just add those pages again and I'm fine with that., but, let me know if there's something I'm missing as far as reverting commits.  

thanks!

**Review by steveoh — APPROVED** [review]

**Pull request merged**